src/IntervalSnapshots.ts

```diff
@@ -112,14 +111,19 @@ function getNumberOfSecondsInInterval(interval: SnapshotInterval): bigint {
   }
 }
 
-// Function to return the interval ID for a given snapshot entity
 function getIdForEntityByInterval(
```

Comment:
@moose-code My only question is if this update works. Previously with the `lastUpdatedTimestamp` being a bigint, there was this computation being done on intervals. I'm wondering if updating this code to convert the date into a unix time and continuing the same computation will work.